GoogleDataTransport/GDTCORLibrary/GDTCORTransformer.m

```diff
@@ -73,35 +73,17 @@ - (void)transformEvent:(GDTCOREvent *)event
       }
     }
     [self.storageInstance storeEvent:transformedEvent];
-    if (self->_runningInBackground) {
-      [[GDTCORApplication sharedApplication] endBackgroundTask:bgID];
-      bgID = GDTCORBackgroundIdentifierInvalid;
-    }
+    [[GDTCORApplication sharedApplication] endBackgroundTask:bgID];
+    bgID = GDTCORBackgroundIdentifierInvalid;
   });
 }
 
 #pragma mark - GDTCORLifecycleProtocol
 
 - (void)appWillForeground:(GDTCORApplication *)app {
-  dispatch_async(_eventWritingQueue, ^{
```

Comment:
you can safely delete these methods if they're empty.

Reply:
SG, thanks.